### PR TITLE
Add custom policy guidelines to integration

### PR DESCRIPTION
**Pull request by mikeurbanski1**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

Inserts guidelines from the platform when pulling custom checks.

There is supposed to be an API call that returns guidelines for custom checks, but it is not doing so. This works around that more reliably.

### CODE REVIEW

**Review by kartikp10 — APPROVED** [review]
Looks good! Pretty sure UTs won't break

**Review by schosterbarak — CHANGES_REQUESTED** [review]
@mikeurbanski1 please add UT under integration tests directory . We have ones that are running with API key

File: integration_tests/test_checkov_cli_integration_report.py [hunk 1]
```diff
@@ -8,6 +9,15 @@
 
 class TestCheckovJsonReport(unittest.TestCase):
 
+    def test_guideline_in_custom_policy(self):
```
Comment (nimrodkor) [inline integration_tests/test_checkov_cli_integration_report.py:12]: @mikeurbanski1 This test should run only when we have the API key, which is embodied in this:
```python
if sys.version_info[1] == 7 and platform.system() == 'Linux':
```

**Comment by nimrodkor** [comment]
@mikeurbanski1 is this still needed?

**Pull request closed without merging**